apps/api/src/coscrad-cli/data-restore.cli-command.e2e.spec.ts

```diff
@@ -109,6 +109,10 @@ describe(`CLI Command: **data-restore**`, () => {
         await testRepositoryProvider.testTeardown();
     });
 
+    afterAll(() => {
+        global.gc && global.gc();
```

Comment:
Let's remove this. We're not running with the `--expose-gc` flag any more, so this won't actually ever trigger the gc() anyway.